recipes/ms-gsl/all/conanfile.py

```diff
@@ -52,3 +54,5 @@ def package_id(self):
         
     def package_info(self):
         self.cpp_info.defines = [self._contract_map[str(self.options.on_contract_violation)]]
```

Comment:
From [release notes](https://github.com/microsoft/GSL/releases/tag/v3.0.0)
> Contract violations are no longer configurable. Contract violations always result in termination, rather than providing a compile-time option to throw or disregard the contract violation. This is subject to change in the future. Some concerns over this decision have been raised and the conversation continues here: isocpp/CppCoreGuidelines#1561. As a side note, the removal of the throwing behavior required the migration of our test infrastructure from Catch2 to Google Test, whose support of death tests easily enabled testing of contract violation behavior.

These options should not be available